rir/src/compiler/native/builtins.cpp

```diff
@@ -884,18 +889,25 @@ void deoptImpl(rir::Code* c, SEXP cls, DeoptMetadata* m, R_bcstack_t* args,
                         cmp.optimizeModule();
 
                         fun = backend.getOrCompile(cnt);
-                        Protect p(fun->container());
+                        PROTECT(fun->container());
```

Comment:
Missing `UNPROTECT(1)`?

Reply:
oh nevermind, findcontext takes care of it..

Reply:
yeah, indeed. but it is bound to cause issues at some point down the line... So I'll at least add a comment or something...